gobblin-data-management/src/main/java/gobblin/data/management/retention/dataset/MultiVersionCleanableDatasetBase.java

```diff
@@ -100,29 +94,73 @@
     implements CleanableDataset, FileSystemDataset {
 
   public static final String CONFIGURATION_KEY_PREFIX = "gobblin.retention.";
+
+  /**
+   * @deprecated in favor of {@link FsCleanableHelper}
+   */
+  @Deprecated
```

Comment:
Why not removing these keys since they share the same name with those in `FsCleanableHelper`?